luaui/RmlWidgets/rml_starter/generate-widget.sh

```diff
@@ -0,0 +1,350 @@
+#!/bin/bash
```

Comment:
So i'm imagining the 0-experience developer starting. Maybe, can we move this widget generation to lua and let them generate a new widget from in the game? 

Not for this PR, but wanted to make the suggestion anyway.

Reply:
If this is possible then definitely, I wasn't totally locked in on how best to create files from in game and whether that had the potential to get out of hand. If it is possible I could even include something like this in a widget builder.